### ISSUE TITLE
Bug: 500 error in history view following meeting time change

### ISSUE THREAD

**Issue by nigel-bmlt** [bmlt-enabled/bmlt-root-server#863]
Server version 3.0.1

Meeting time was changed via new admin API using bmlt workflow plugin.

Laravel reports the following for this meeting when viewing the history:
```
[previous exception] [object] (TypeError(code: 0): implode(): Argument #2 ($array) must be of type ?array, string given at /home/naorgau/public_html/main_server/legacy/local_server/server_admin/c_comdef_admin_ajax_handler.class.php:1052)
[stacktrace]
#0 /home/naorgau/public_html/main_server/legacy/local_server/server_admin/c_comdef_admin_ajax_handler.class.php(1052): implode()
#1 /home/naorgau/public_html/main_server/legacy/local_server/server_admin/c_comdef_admin_ajax_handler.class.php(93): c_comdef_admin_ajax_handler->GetMeetingHistory()
#2 /home/naorgau/public_html/main_server/legacy/local_server/server_admin/c_comdef_admin_ajax_handler.class.php(1522): c_comdef_admin_ajax_handler->parse_ajax_call()
#3 /home/naorgau/public_html/main_server/legacy/local_server/index.php(43): require_once('/home/naorgau/p...')
#4 /home/naorgau/public_html/main_server/legacy/index.php(21): include('/home/naorgau/p...')
#5 /home/naorgau/public_html/main_server/storage/framework/views/3c3f7cc27ab1ca049bdd9c908ea5eacbad4c386c.php(1): require_once('/home/naorgau/p...')
#6 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(109): require('/home/naorgau/p...')
#7 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(110): Illuminate\\Filesystem\\Filesystem::Illuminate\\Filesystem\\{closure}()
#8 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\\Filesystem\\Filesystem->getRequire()
#9 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(70): Illuminate\\View\\Engines\\PhpEngine->evaluatePath()
#10 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/View/View.php(152): Illuminate\\View\\Engines\\CompilerEngine->get()
#11 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/View/View.php(135): Illuminate\\View\\View->getContents()
#12 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/View/View.php(104): Illuminate\\View\\View->renderContents()
#13 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Http/Response.php(69): Illuminate\\View\\View->render()
#14 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\\Http\\Response->setContent()
#15 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(57): Illuminate\\Http\\Response->__construct()
#16 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(87): Illuminate\\Routing\\ResponseFactory->make()
#17 /home/naorgau/public_html/main_server/app/Http/Controllers/CatchAllController.php(82): Illuminate\\Routing\\ResponseFactory->view()
#18 /home/naorgau/public_html/main_server/app/Http/Controllers/CatchAllController.php(47): App\\Http\\Controllers\\CatchAllController::legacyResponse()
#19 /home/naorgau/public_html/main_server/app/Http/Controllers/Legacy/LegacyAuthController.php(62): App\\Http\\Controllers\\CatchAllController::handle()
#20 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Http\\Controllers\\Legacy\\LegacyAuthController->handle()
#21 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(43): Illuminate\\Routing\\Controller->callAction()
#22 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Route.php(260): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#23 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
#24 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Router.php(798): Illuminate\\Routing\\Route->run()
#25 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#26 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#27 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
#28 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Session/Middleware/AuthenticateSession.php(59): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#29 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\AuthenticateSession->handle()
#30 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#31 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#32 /home/naorgau/public_html/main_server/app/Http/Middleware/StartSession.php(23): Illuminate\\Session\\Middleware\\StartSession->handle()
#33 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): App\\Http\\Middleware\\StartSession->handle()
#34 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#35 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#36 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#37 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#38 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#39 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Router.php(799): Illuminate\\Pipeline\\Pipeline->then()
#40 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Router.php(776): Illuminate\\Routing\\Router->runRouteWithinStack()
#41 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Router.php(740): Illuminate\\Routing\\Router->runRoute()
#42 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Routing/Router.php(729): Illuminate\\Routing\\Router->dispatchToRoute()
#43 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(190): Illuminate\\Routing\\Router->dispatch()
#44 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#45 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#46 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31):Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#47 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#48 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#49 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#50 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#51 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#52 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#53 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#54 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#55 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php(62): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#56 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\HandleCors->handle()
#57 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#58 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#59 /home/naorgau/public_html/main_server/app/Http/Middleware/DatabaseMigrations.php(46): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#60 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): App\\Http\\Middleware\\DatabaseMigrations->handle()
#61 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#62 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(165): Illuminate\\Pipeline\\Pipeline->then()
#63 /home/naorgau/public_html/main_server/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(134): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#64 /home/naorgau/public_html/main_server/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle()
#65 {main}
"}
```

Digging as best I can into the change history, the following entries are recorded for meeting ID 3191 

[change record.txt](https://github.com/bmlt-enabled/bmlt-root-server/files/10479136/change.record.txt)

**Comment by jbraswell** [comment]
I think https://github.com/bmlt-enabled/bmlt-root-server/pull/864 fixes this specific error, but what I am more concerned about it is why there might be two change records with no differences. There is either something wrong with the UI code that generates the change description, or the rest api code that inserts the change records. Could you send me a copy of your database on Slack?

**Comment by jbraswell** [comment]
Looks like this was fixed by #864, and is released in 3.0.2.
